pyccel/ast/builtin_methods/set_methods.py

```diff
@@ -0,0 +1,75 @@
+# coding: utf-8
+#------------------------------------------------------------------------------------------#
+# This file is part of Pyccel which is released under MIT License. See the LICENSE file or #
+# go to https://github.com/pyccel/pyccel/blob/master/LICENSE for full license details.     #
+#------------------------------------------------------------------------------------------#
+"""
+The Set container has a number of built-in methods that are 
+always available.
+
+This module contains objects which describe these methods within Pyccel's AST.
+"""
+from pyccel.ast.datatypes import NativeVoid, NativeGeneric
+from pyccel.ast.internals import PyccelInternalFunction
+from pyccel.ast.builtins import PythonTuple
+
+__all__ = ('SetAdd',)
+
+class SetAdd(PyccelInternalFunction) :
+    """
+    Represents a call to the .add() method.
+    
+    Represents a call to the .add() method which add an element
+    to the set if it's not already present.
+    If the element is already in the set, the set remains unchanged.
+    Parameters
```

Comment:
```suggestion
    If the element is already in the set, the set remains unchanged.

    Parameters
```

Reply:
Numpydoc cannot detect the parameters section without this space

Reply:
fixed : d7839d8